Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.cpp

```diff
@@ -398,6 +398,8 @@ void CoordinatedGraphicsLayer::setContentsToPlatformLayer(PlatformLayer* platfor
         if (m_platformLayer)
             m_platformLayer->setClient(nullptr);
         m_shouldSyncPlatformLayer = true;
+        if (platformLayer)
+            m_shouldUpdatePlatformLayer = true;
```

Comment:
I think it would be better to just call m_platformLayer->swapBuffersIfNeeded() inside syncPlatformLayer(), just after storing the proxy, so we are sure that it will be called just once when setting a new PlatformLayer. Other than that looks ok.

Reply:
@magomez, hi. I'm a bit confused with this. `m_platformLayer->swapBuffersIfNeeded()` is still called from `CoordinatedGraphicsLayer::updatePlatformLayer()`, so it is not clear what you trying to achive here. Do you mean to make the intention more explicit by calling `m_platformLayer->swapBuffersIfNeeded()` directly from `CoordinatedGraphicsLayer::syncPlatformLayer()`?

Reply:
Yeah, but nevermind, it was just a cosmetic issue.